crypto/rsa/rsa_sp800_56b_gen.c

```diff
@@ -392,6 +393,15 @@ int ossl_rsa_sp800_56b_generate_key(RSA *rsa, int nbits, const BIGNUM *efixed,
         /* (Step 2) Generate prime factors */
         if (!ossl_rsa_fips186_4_gen_prob_primes(rsa, info, nbits, e, ctx, cb))
             goto err;
+
+        if (!info){
```

Comment:
There should be a space between the `)` and the `{`, but why have two `if()`s rather than one using `&&`?

Reply:
Willl make the changes.